testsuite/integration/clustering/pom.xml

```diff
@@ -438,5 +438,232 @@
+                                            </layers>
+                                        </config>
+                                        <!-- Some tests launch an lb process, and this gets set up
+                                             by ts.config-as.clustering copying the single common dist
+                                             to a separate dir, and then the tests launch a process from
+                                             that dir with -c=standalone-load-balancer.xml. We could
+                                             directly provision to that dir in a separate execution and
+                                             then change the ts.config-as.clustering copying, which would be
+                                             more formally correct. But this is simpler for now and, at least
+                                             right now, does not result in extra packages ending up in the
+                                             dist we test. (Avoiding the risk of extra packages would be the
+                                             reason to do a separate provisioning of this installation.) -->
+                                        <config>
+                                            <model>standalone</model>
+                                            <name>standalone-load-balancer.xml</name>
+                                            <layers>
+                                                <layer>cloud-server</layer>
+                                                <layer>undertow-load-balancer</layer>
+                                            </layers>
+                                        </config>
+                                        <!-- Produce a standalone.xml too, but with the same slimmed ha content,
+                                             not the standard standalone.xml content in the full dist.
+                                             The tests in the ts.surefire.clustering.single-node execution
+                                             expect a server with this config file to be present. If it
+                                             turns out running these tests with this set of layers doesn't really
+                                             make sense we can revisit.
+                                        <config>
+                                            <model>standalone</model>
+                                            <name>standalone.xml</name>
+                                            <layers>
+                                                <layer>cloud-server</layer>
+                                                <layer>web-clustering</layer>
+                                                <layer>h2-default-datasource</layer>
+                                            </layers>
+                                        </config>
+                                         -->
```

Comment:
Should this be removed or is it a marker for future?

Reply:
I assume the intention is to keep it ... but I don't see where the `standalone.xml` is provisioned to run the "single node" tests? (Actually, that seems to be a misnomer, it should be more like "unclustered").

Reply:
Keep in mind this is only done if the `ts.layers` property is set.

Reply:
The intent is to remove it, which I'll do in a sec. None of the single node tests work and are meaningful with this slimmed server.

Reply:
Also if we wanted any single node tests to run this way we'd need to provision a different server, not just another config as 1) the point is to use a non-HA config for those, so different config content and 2) the server we use for the HA tests is meant to be slimmed to the packages needed for HA, and shouldn't include things that a non-HA config might bring in. So, it would be an entirely separate provisioning execution.